plugins/catalog-backend/src/modules/core/FileReaderProcessor.ts

```diff
@@ -56,11 +58,16 @@ export class FileReaderProcessor implements CatalogProcessor {
           for await (const parseResult of parser({
             data: data,
             location: {
-              type: 'file',
+              type: LOCATION_TYPE,
               target: path.normalize(fileMatch),
             },
           })) {
             emit(parseResult);
+            emit(
+              processingResult.refresh(
+                `${LOCATION_TYPE}:${path.normalize(fileMatch)}`,
```

Comment:
```suggestion
                `${LOCATION_TYPE}://${path.normalize(fileMatch)}`,
```

Are these always absolute paths btw? I'm thinking that would likely be best

Reply:
Based on my testing the npm glob library will return the absolute paths in this case, so here the fileMatch will always be absolute.